technologies/app/rstudio-4.1/rstudio-4.1/Dockerfile

```diff
@@ -0,0 +1,101 @@
+FROM rocker/tidyverse:4.1.1
```

Comment:
Why not having a FROM Saagie R image (most of the  content of the Dockerfile between R and Rstudio are identical) and adding nginw and the correct entrypoint for Rstudio?